src/run_docker.py

```diff
@@ -157,32 +156,24 @@ async def sleep_until_next_polling(current_polling_duration_s):
         await sleep_until_next_polling(polling_duration)
 
 
-def run_loop_forever():
-    try:
-        loop.run_forever()
-    finally:
-        print("Closing AsyncIO loop...")
-        webserver.close_and_cleanup(loop)
-        loop.close()
-
-
 def main():
+    threading.Thread(target=run_webserver_on_asyncio_loop_forever,
+                     args=(loop,),
```

Comment:
Why share your loop? your = from this thread

Reply:
Created new loop in thread